tensorflow/python/kernel_tests/math_ops/bincount_op_test.py

```diff
@@ -178,6 +178,22 @@ def test_invalid_inputs(self):
               weights=weights,
               binary_output=binary_output))
 
+  def test_negative_value(self):
+    with self.assertRaises(errors.InvalidArgumentError):
+      arg_0_tensor = random_ops.random_uniform(
```

Comment:
nit: arg_0 -> value, as other args are not using "arg_x" format

Reply:
@vufg Updated.